compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -655,6 +655,12 @@ error.invalid.namespace.declaration=\
 error.cannot.update.xml.sequence=\
   cannot update an xml sequence
 
+error.operation.does.not.support.xml.navigation.access=\
+  invalid operation: type ''{0}'' does not support xml navigation access
```

Comment:
Shall we say "union type xxxxxxxx does not support ....." as this only applicable to unions and I think it's easier for the user to understand